pyproject.toml

```diff
@@ -115,6 +107,39 @@ lint = [
 calibration = [
   "scipy>=1.14.0",
 ]
+plugin_cohere = [
+  "cohere>=4.5.1,<5"
+]
+plugin_langchain = [
+  "langchain>=0.0.300",
+]
+plugin_llava = [
+  "PIL",
+]
+plugin_litellm = [
+  "litellm>=1.41.21,<1.67.2",
+]
+plugin_mistralai = [
+  "mistralai==1.5.2",
+]
+plugin_nemoguardrails = [
+  "nemoguardrails",
+]
+plugin_nemollm = [
+  "nemollm>=0.3.0",
+]
+plugin_octoai = [
+  "octoai-sdk>=0.8.0",
+]
+plugin_ollama = [
+  "ollama>=0.4.7",
+]
+plugin_optimum = [
+  "optimum-nvidia",
+]
+plugin_replicate = [
+  "replicate>=0.8.3",
+]
```

Comment:
Consider adding to allows for `pip install garak[plugin_all]`:
```
plugin_all = garak[
    plugin_cohere,
    plugin_langchain,
    plugin_llava,
    plugin_litellm,
    plugin_mistralai,
    plugin_nemoguardrails,
    plugin_nemollm,
    plugin_octoai,
    plugin_ollama,
    plugin_replicate,
]
```

There might be a better name to use for this group.

Reply:
This seems no worse than status quo, except with optional installation, two plugins with conflicts can now co-exist as descriptions